application/rtthread/msh/main.c

```diff
@@ -12,6 +12,13 @@
 #include "hbird_sdk_soc.h"
 #include <rtthread.h>
 
+void hbird(int argc, char* argv[])
```

Comment:
Any better name?

Reply:
I tried *hello* which is much better, unfortunately, it has been used somewhere else in rt-thread. So I just pick up hbird-sdk.